test/bouture.test.js

```diff
@@ -267,3 +267,29 @@ describe(`Bouture.ul( ['a', 'b', 'c'].map(text => Bouture.li(text)))`, () => {
     expect(ul.children[0].textContent).to.equal('a')
   })
 })
+
+describe(`Bouture.button({onClick: e => { called = true }})`, () => {
+  it('should create a button and bind with an onclick event', () => {
+    let called = true
```

Comment:
This should probably be initialized to false ;-)

Reply:
I had the impression of having found a simplified way to pass the tests ;)